pages/index.js

```diff
@@ -1,63 +1,149 @@
-            </Row>
+      {/* Hero Banner */}
+      <div className={`${styles.heroBanner}`}>
+        <div className={`${styles.heroGrid}`}>
+          <div className={`${styles.heroGridItem}`}>
+            <p className={`${styles.heroIcon1}`}>Enhance</p>
+            <p className={`${styles.heroIcon2}`}>Empower</p>
+            <p className={`${styles.heroIcon3}`}>Enable</p>
+            <img 
+              src='/assets/img/graphics/anteater_laptop.png'
+              height="500px"
+              alt="Anteater On Laptop"
+              />
+            </div>
+            {/* Terminal Container Graphic Inspired from the CalHacks Team at Berkeley */}
+            <div className={`${styles.heroGridItemSplash}`}>
+              <div className={`${styles.heroSplashContainer}`}>
+                <h1>
+                  ICS Student Council
+                </h1>
+                <p>
+                  Improving ICS students' lives professionally, socially, and academically
+                </p>
+                <Button type="button" className={`${styles.heroButton}`} href="/get-involved">Get Involved ></Button>
+              </div>
+            </div>
+        </div>
+      </div>
+      
+      {/* About Section */}
+      {/* Wave Graphic From the CalHack Team at Berkeley! */}
+      <img className={`${styles.heroFooterBackground}`} src="/assets/img/graphics/wave.png" alt="Box Edge"/>
+      <div className={`${styles.heroFooter}`}>
+        <div className={`${styles.heroFooterContent}`}>
+          <div className={`${styles.heroFooterText}`}>
+            <h1>About Us</h1>
+            <p>ICS Student Council, founded in 2010, is a student-run organization that serves ICS students. Our mission is to deepen students' connections with the Donald Bren School of Information and Computer Sciences & UCI Alumni and improve them professionally, socially, and academically.</p>
+            <a href="/get-involved">Learn More ></a>
+          </div>
+          <img src='/assets/img/graphics/anteater_hw.png' alt="About Us Graphic" />
+        </div>
+      </div>
+
+      {/* Events Section */}
+      <div className={"sectionAlt"}>
+        <Row>
+          <Col lg={12} sm={12}>
+            <h2>Our Events</h2>
+            <p>We aim to host events that will better ICS Students academically, professionally, and socially. <br></br>Here are some moments from our previous events!</p>
           </Col>
         </Row>
-      </Container>
+        <div className={`${styles.eventBox}`}>
+          <Carousel prevLabel={'Back'} nextLabel={'Next'} indicators={false} fade={true} slide={false} touch={false}>
+          {EventsData["past_pictures"].map((past, ind) => (
+            <Carousel.Item key={ind}>
+              <img
+                className={`d-block`}
+                src={past.src}
+                alt={past.title}
+              />
+            </Carousel.Item>
+          ))}
+          </Carousel>
```

Comment:
"Next" and "Back" are pretty hard to read. Idk the best way to fix that though. Maybe a background shadow or something? Not a huge deal if you leave it as is though.
<img width="542" alt="image" src="https://user-images.githubusercontent.com/29494270/199140770-f1516758-83eb-4e9d-bf63-fef70484ffac.png">

Reply:
Yeah, I'll leave it as it is for now. I added it to make it accessible to people using a website reader.

Reply:
+1 to this

Reply:
I ended up removing the text D: I don't think it makes a difference for a website reader to know the arrows.